backend/src/tracking/tracking/tracking.service.ts

```diff
@@ -667,7 +667,12 @@ export class ParcelLabTrackingService {
       recipient: this.getName(shopifyOrder),
       recipient_notification: this.getName(shopifyOrder),
       statuslink: shopifyOrder?.order_status_url,
-      street: shopifyOrder?.shipping_address?.address1 + (shopifyOrder?.shipping_address?.address2?.trim() ? `\n${shopifyOrder?.shipping_address?.address2}` : ''),
+      street: shopifyOrder?.shipping_address?.address1
```

Comment:
Can you create a method `getStreet` for this? Because you do the same thing on two different parts. See `this.getName` above.